service/src/test/java/bio/terra/axonserver/app/controller/CromwellWorkflowControllerTest.java

```diff
@@ -0,0 +1,142 @@
+package bio.terra.axonserver.app.controller;
+
+import static bio.terra.axonserver.testutils.MockMvcUtils.USER_REQUEST;
+import static bio.terra.axonserver.testutils.MockMvcUtils.addAuth;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+
+import bio.terra.axonserver.service.cromwellworkflow.CromwellWorkflowService;
+import bio.terra.axonserver.service.wsm.WorkspaceManagerService;
+import bio.terra.axonserver.testutils.BaseUnitTest;
+import io.swagger.client.model.CromwellApiLabelsResponse;
+import io.swagger.client.model.CromwellApiWorkflowIdAndStatus;
+import io.swagger.client.model.CromwellApiWorkflowMetadataResponse;
+import io.swagger.client.model.CromwellApiWorkflowQueryResponse;
+import io.swagger.client.model.CromwellApiWorkflowQueryResult;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+public class CromwellWorkflowControllerTest extends BaseUnitTest {
```

Comment:
let's test for validation failure too

Reply:
@yuhuyoyo I updated the tests a bit. How do they look? 

Mainly concerned about the validation failure test.